schema.proto

```diff
@@ -152,7 +152,7 @@ message OpUnary {
   }
 
   required Kind kind = 1;
-  optional string ffiName = 2;
+  optional uint64 ffiName = 2;
```

Comment:
the spec should be updated to indicate that the name is interned

Reply:
done